sky/setup_files/setup.py

```diff
@@ -87,7 +101,15 @@
         'Programming Language :: Python :: 3.7',
         'Programming Language :: Python :: 3.8',
         'Programming Language :: Python :: 3.9',
+        'License :: OSI Approved :: Apache Software License',
+        'Operating System :: OS Independent',
+        'Topic :: Software Development :: Libraries :: Python Modules',
+        'Topic :: System :: Distributed Computing',
```

Comment:
plus cloud computing too?

Reply:
Yeah, seems `Distributed Computing` is the closest one.